### PR TITLE
Improvement: Compact Dicer Tracker

### PR DESCRIPTION
**Pull request by Jordyrat**
## What
Shortened the Dicer Tracker and added an option for a compacted version.

<details>
<summary>Images</summary>

image
image

</details>

## Changelog Improvements
+ Added an option for a compacted Dicer RNG Drop Tracker Display. - Jordyrat
    * Also shortened the default display.

### CODE REVIEW

**Comment by Thunderblade73** [comment]
The changelog shouldn't be two entries here.

**Comment by Jordyrat** [comment]
> The changelog shouldn't be two entries here.

Updated; wasn't sure since it's 2 changes.

File: src/main/java/at/hannibal2/skyhanni/features/garden/farming/DicerRngDropTracker.kt [hunk 1]
```diff
@@ -109,13 +109,31 @@ object DicerRngDropTracker {
...
+    private fun drawDisplay(data: Data) = buildList {
         val cropInHand = cropInHand ?: return@buildList
         val items = data.drops.getOrPut(cropInHand) { mutableMapOf() }
-        addAsSingletonList("§7Dicer RNG Drop Tracker for $toolName§7:")
-        for ((rarity, amount) in items.sortedDesc()) {
-            val displayName = rarity.displayName
-            addAsSingletonList(" §7- §e${amount.addSeparators()}x $displayName")
+        val list = mutableListOf<Renderable>()
+        val topLine = mutableListOf<Renderable>()
+        var compactLine = ""
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/garden/farming/DicerRngDropTracker.kt]: this should be inside the if statement where it is used

File: src/main/java/at/hannibal2/skyhanni/features/garden/farming/DicerRngDropTracker.kt
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/features/garden/farming/DicerRngDropTracker.kt:1]: also add a few line breaks in drawDisplay to make it more readable

**Review by CalMWolfs — CHANGES_REQUESTED** [review]

**Review by CalMWolfs — APPROVED** [review]

**Pull request merged**